content/latest/api/ycql/type_jsonb.md

```diff
@@ -197,6 +197,38 @@ cqlsh> SELECT * FROM store.books WHERE id = 4;
   4 | {"author":{"first_name":"John","last_name":"Doe"},"editors":["Robert","Jack","Melisa"],"genre":"novel","name":"Great Expectations","year":1950}
 ```
 
+- Update a missing JSONB subdocument resulting in an insert.
+
+```sql
+cqlsh> UPDATE store.books SET details->'publisher' = '"Chapman and Hall"' WHERE id = 4;
+```
+
+```sql
+cqlsh> SELECT * FROM store.books WHERE id = 4;
+```
+
+```
+ id | details
+----+-------------------------------------------------------------------------------------------------------------------------------------------------
+  4 | {"author":{"first_name":"John","last_name":"Doe"},"editors":["Robert","Jack","Melisa"],"genre":"novel","name":"Great Expectations","publisher":"Chapman and Hall","year":1950}
+```
```

Comment:
Also add two more examples:
1.  Where details={} and then you run `UPDATE store.books SET details->'publisher'='some publisher'`
2. Where value being inserted is a JSON object - something like `UPDATE store.books SET details->'newfield'='{"a":"100", "b":"200"}' WHERE id = 1;`